test/basic.js

```diff
@@ -79,10 +79,10 @@ t.test('find when executable', function (t) {
     process.env.PATHEXT = '.SH'
 
     t.test('foo.sh', function (t) {
-      runTest('foo.sh', t)
+      runTest(fixture + '/foo.sh', t)
```

Comment:
Doesn't this sort of defeat the purpose of using `which`?  The point is that `fixture` is in the PATH env, so finding just `foo` alone should work.